proto/proto/requests.proto

```diff
@@ -215,3 +215,24 @@ message GetUnconsumedNetworkNotesRequest {
     // Number of notes to retrieve per page.
     uint64 page_size = 2;
 }
+
+// Returns a list of unconsumed network notes for a network account using pagination.
+message GetUnconsumedNetworkNotesForNetworkAccountRequest {
+    // An opaque token used to paginate through the notes.
+    //
+    // This should be null on the first call, and set to the response
+    // token until the response token is null, at which point all data
+    // has been fetched.
+    optional uint64 page_token = 1;
+
+    // Number of notes to retrieve per page.
+    uint64 page_size = 2;
+
+    // The network account ID prefix to filter notes by.
+    uint32 network_account_id_prefix = 3;
+
+    // The maximum (inclusive) block number to scope the query by.
+    // Notes that are not consumed before or within the specified block number
+    // are treated as unconsumed.
+    fixed32 block_num = 4;
```

Comment:
Trying to think of a better name here; currently context is only known if reading  the comment.

But I don't really have anything.. `unconsumed_at`, `state_at`,..

Reply:
Went with
```proto
     // The maximum (inclusive) block number to scope the query by.
     //
     // Notes that are consumed after this block number are excluded from the result.
     fixed32 latest_block_num = 4;
```